run/gem5art/run.py

```diff
@@ -472,8 +472,11 @@ def saveResults(self) -> None:
 
         with zipfile.ZipFile(os.path.join(self.outdir, 'results.zip'), 'w',
                              zipfile.ZIP_DEFLATED) as zipf:
-            for f in files:
-                zipf.write(os.path.join(self.outdir, f))
+            for root, dirs, files in os.walk(self.relative_outdir):
+                if root == self.relative_outdir:
```

Comment:
This won't merge cleanly with the changeset removing the relative outdir... What will need to be done to fix this?

Reply:
So when we create the gem5Run object, we must give `outdir` an abs path?

Reply:
Yes. See https://github.com/darchr/gem5art/pull/50 and https://github.com/darchr/gem5art/pull/50/commits/9a139d613e8c11ab4ace0647ec26d75d05607a13.

I don't think there's any way around that. However, there will be a `cwd` in run (https://github.com/darchr/gem5art/pull/50/commits/9a139d613e8c11ab4ace0647ec26d75d05607a13#diff-7df16e1be50255dbdcbdaefa049d0cc3R379) if you wanted to use a directory relative to "results".

However, it's probably better not to zip based on the "results" directory, but to just zip the files in the output. So, if the output dir is `/home/jlp/gem5/experiment/results/linux/X86/cooltest/config1/` I would simply zip relative to `config1/`

Reply:
So we can find the first folder that has a file, and zip everything relative to that folder?

Reply:
Oh, we can simply look for the subdirectory containing simout / simerr files and zip everything relative to that folder.